docs/04-operation-guides/operations/50-kiali-grafana.md

```diff
@@ -36,29 +36,36 @@ By default, Kyma does not expose Kiali and Grafana. However, you can still acces
 
-To make Kiali and Grafana permanently accessible, expose the services securely using [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/).
+Kyma manages an [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/) instance to secure access to Kiali, Grafana and Jaeger. To make the services accessible, configure OAuth2 Proxy by create a Kubernetes `Secret` with your identity provider credentials.
 
 ### Steps
 
-The following example shows how to use Github as authentication provider for Kiali and Grafana. You create an `oauth2_proxy` `Deployment` to achieve this, and expose it as a `VirtualService` via Kyma's Istio Gateway.
-
->**NOTE:** The `oauth2_proxy` supports a wide range of other well-known authentication services or OpenID Connect for custom solutions. See the [list of supported providers](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider) to find instructions for other authentication services.
+The following example shows how to use an OpenID Connect (OIDC) compliant identity provider for Kiali, Grafana and Jaeger.
 
-1. Chose a domain for the exposed service under the Kyma base domain. For example, if your Kyma cluster is reachable under `kyma.example.com`, use `kiali.kyma.example.com` or `grafana.kyma.example.com`, respectively.
+>**NOTE:** The OAuth2 Proxy supports a wide range of other well-known authentication services or OpenID Connect for custom solutions. See the [list of supported providers](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider) to find instructions for other authentication services.
 
-2. Create a new Github application under https://github.com/settings/apps. Set the callback URL to `https://kiali.kyma.example.com/oauth2/callback`. Ensure at least read-only access to email addresses for the Github application. Copy the client ID and secret.
+1. Create a new OpenID Connect application for your identity provider. Set the callback URL to the `/oauth2/callback` path of your service. For instance `https://kiali.kyma.example.com/oauth2/callback`. Your identity provider will return a client ID, a client secret and a token issuer URL.
 
-3. Create a Kubernetes Secret for the client ID and secret:
+2. Create a `Secret` for the OAuth2 Proxy configuration [environment variables](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview/#environment-variables). For an OpenID Connect compliant provider, adapt the client ID, secret and token issuer to the values that were provided while creating the application. To limit access to specific user groups, configure this with the `OAUTH2_PROXY_ALLOWED_GROUPS` variable and ensure that `OAUTH2_PROXY_OIDC_GROUPS_CLAIM` points to the groups attribute name that is used by your authentication service (`groups` is the default). See [OAuth2 Proxy docs](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider/) to get the required configuration flags for other identity provider types:
```

Comment:
```suggestion
2. Create a `Secret` for the OAuth2 Proxy configuration [environment variables](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview/#environment-variables). 
  - For an OpenID Connect compliant provider, adapt the client ID, secret and token issuer to the values that were provided while creating the application. 
    - To limit access to specific user groups, configure this with the `OAUTH2_PROXY_ALLOWED_GROUPS` variable and ensure that `OAUTH2_PROXY_OIDC_GROUPS_CLAIM` points to the groups attribute name that is used by your authentication service (`groups` is the default). To get the configuration flags required for other identity provider types, see [OAuth2 Proxy docs](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/oauth_provider/).
```

Not sure if I got the formatting right, but basically I'd suggest to break down the long paragraph into bullets within the step (that is, below this numbered list item).